addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -285,6 +285,20 @@ registerModel({
             }
             return this.isAddingChat;
         },
+        /**
+         * @private
+         * @returns {MobileMessagingNavbarView|FieldCommand}
```

Comment:
When it's a relation field it's not an `or`, it's always just a `FieldCommand`.
`|` is only for attributes when returning either a value or a command (usually `clear`).